kie-server-parent/kie-server-services/kie-server-services-kafka/src/main/java/org/kie/server/services/jbpm/kafka/KafkaServerExtension.java

```diff
@@ -301,18 +339,13 @@ private void unsubscribe() {
     }
 
     private static <T extends SignalDescBase> String topicFromSignal(T signal) {
-        return System.getProperty(TOPIC_PREFIX + signal.getName(), signal.getName());
+        return topicFromSignal(signal.getName());
     }
-
-    @Override
-    public void serverStarted() {
-        // will use lazy initialization for consumer
+    
+    private static <T extends SignalDescBase> String topicFromSignal(String name) {
```

Comment:
In this method, T is not used, you can remove the generics
```suggestion
    private static String topicFromSignal(String name) {
```